### DISCUSSION TITLE
Nightly builds

### DISCUSSION THREAD

**Discussion by vascoferraz** [provectus/kafka-ui#3790] [Q&A] [answered]
Hi everyone :)

Do you publish nightly builds?

If so, where?

Thanks.

**Comment by Haarolean** [comment, marked as answer]
Hi, every each commit on master is being built and published to [docker hub](https://hub.docker.com/r/provectuslabs/kafka-ui)

**Reply by vascoferraz** [reply]
Thanks :)
